index.html

```diff
@@ -15,29 +30,26 @@
 
       <div id='blogTitle'>
         <h1>Maria's Blog</h1>
-        <h3>Some super exciting theme goes here</h3>
       </div>
 
       <nav>
         <div class='tabLinks'>
-
-            <a href='#'>Home</a>
-            <a href='#'>About</a>
+            <a href='#'></span>Home</a>
+            <a href='#'></span>About</a>
```

Comment:
It's strange to have a closing `</span>` tag, but not an opening one. Does the opening one come from JavaScript?